tico/serialize/operators/op_avg_pool2d.py

```diff
@@ -142,11 +141,7 @@ def define_padding_node():
                 ],
                 dtype=torch.int32,
             )
-            input_shape, input_shape_signature = extract_circle_shape(input)
-
-            if input_shape_signature is not None:
-                raise RuntimeError("Dynamic shape is not supported yet.")
-
+            input_shape = extract_shape(input)
```

Comment:
Just curiosity, H or W can be a dynamic dimension, too?

Reply:
Ah, I asked this because I wondered if arithmetic operation can be done with symbolic values like below. Most of cases will be a batch dimension though.

```python
padded_input_shape[1] += padding[0] * 2
```

Reply:
@mhs4670go Yes, it is possible. `torch.SymInt support all dimensions to be symbolic integer in this code.` -> Symbolic Integers are able to compute arithmetic operations with integers.